docs/source/advanced/backends.rst

```diff
@@ -194,3 +194,7 @@ Complete list of level 2 functions to implement::
     def write(self, session, data):
     def write_asynchronously(self, session, data):
     def write_from_file(self, session, filename, count):
+
+Additionally, a |ResourceManager| attribute check for 
+:class:`pyvisa.highlevel.VisaLibraryBase.open_resource` is implemented 
+in order to support custom opening handling by the new backend implementation.
```

Comment:
I think this would be easier to discover as a `.. note::` directive on line 93 since people checking what they need to implement are likely to be interested in this.

Reply:
Sure if it increase visibility! Just moved it to line 93.